docs/content/en/contribute/general/github/_index.md

```diff
@@ -0,0 +1,276 @@
+---
+title: Working with github
+description: Information relevant to github
+weight: 100
+---
+
+To start your contribution with
+[Keptn Lifecycle Toolkit](https://github.com/keptn/lifecycle-toolkit)
+follow this github instructions:
+
+## Fork and clone the repository
+
+Perform the following steps to create a copy of this repository on your local machine:
+
+1. Fork the Keptn repository:
+
+     * Log into GitHub (or create a GitHub account and then log into it).
+     * Go to the [Keptn lifecycle-toolkit repository](https://github.com/keptn/lifecycle-toolkit).
+     * Click the **Fork** button at the top of the screen.
+     * Choose the user for the fork from the options you are given,
+       usually your GitHub ID.
+
+   A copy of this repository is now available in your GitHub account.
+
+2. Get the string to use when cloning your fork:
+
+     * Click the green "Code" button on the UI page.
+     * Select the protocol to use for this clone (either HTTPS or SSH).
+     * A box is displayed that gives the URL for the selected protocol.
+       Click the icon at the right end of that box to copy that URL.
+
+3. Run the **git clone** command from the shell of a local directory
```

Comment:
Should we mention that they might want to create a sub-directory called "Clone" or "Github" or "Keptn" or whatever they want but to keep all their clones together in one place?  It's not a rule but it is something that can make life easier and novices may not intuit this.